wp-content/themes/redmond/web/scss/app.scss

```diff
@@ -22,3 +22,41 @@
 @import "modules/title-section";
 
 // classes
+h2 {
+  text-transform: uppercase;
```

Comment:
this is the default for all headings. I don't think there is a setting for this so have another selector h1, h2, h3, h4, h5, h6 with this.